renovate.json

```
@@ -31,6 +31,20 @@
             "matchUpdateTypes": [
                 "major"
             ]
+        },
+        {
+            "_comment": "We dont want redis to be updated due to licencing changes",
```

```suggestion
            "_comment": "We dont want redis to be updated due to licensing changes",
```

Lets stick to american english here.